views/registerView.ejs

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Register</title>
+    <link rel='stylesheet' href='/stylesheets/register.css' />
+  </head>
+  <body>
+    <div class="container">
+        <div class="header">
+            <h2>Member Registration</h2>
+        </div>
+        <div class="main">
+          <form method="post">
+            <div class="linput"> <input type="email" placeholder="email"></div><!-- Klasa nazywa sie linput, bo inaczej nazwa bylaby taka sama jak input type co nie? -->
```

Comment:
1. you can use class directly on the input element: `<input type="email" placeholder="email" class="linput">`

2. linput doesn't seem like a good name - try to find something meaningful (e.g. `primary` and `secondary`; maybe something related to it's properties like: `centered`)

Reply:
Okey, I just removed linput class because it had no specific use, just used "input" in it's place